Sources/APLFormContentCreationLayer/DataTransferEntities/DataTransferEntity.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Created by apploft on 07.04.20.
+// Copyright © 2019 apploft GmbH.
+// MIT License · http://choosealicense.com/licenses/mit/
+//
+
+import Foundation
+
+///Content types are used in multipart/form-data requests. Each DataTransferEntity needs a content type to
+/// properly generate the body of the request.
+public enum ContentType: String {
+    case binary = "application/octet-stream"
+    case text = "text/plain"
+    case css = "text/css"
+    case html = "text/html"
+    case javaScript = "text/javascript"
+    case imagePNG = "image/png"
+    case imageJPEG = "image/jpeg"
+    case imageGIF = "image/gif"
+    case imageSVG = "image/svg+xml"
+    case audioMP3 = "audio/mpeg"
+    case videoMP4 = "video/mp4"
+}
+
+/// A class for storing data and its associated propoperties like its content type. This class can be used as a data structure
+/// when creating multipart/form-data or application/x-www-form-urlencoded requests.
+public class DataTransferEntity {
```

Comment:
Sollten wir 'DataTransferEntity' nicht als protocol (aka Interface) definieren?

Reply:
Nein, weil man damit Data Objekte (zb. Data([0x02]) oder eine ausgelesene File) in eine TransferEntity umwandeln kann. Sonst kann man 'nur' Texte und UIImages in eine TransferEntity umwandeln. :)